src/logging.h

```diff
@@ -56,6 +56,7 @@ namespace BCLog {
         QT          = (1 << 19),
         LEVELDB     = (1 << 20),
         VALIDATION  = (1 << 21),
+        FLATFILE    = (1 << 22),
```

Comment:
I am not sure it makes sense to make a new debug category for this.

But if you do, I would prefer something conceptual like "BLOCKSTORAGE", flat file is an implementation detail and fairly irrelevant to users.

Also don't forget updating the table in `logging.cpp` so that it can actually be enabled :slightly_smiling_face:

Reply:
Good points!

Now putting it in `BLOCKSTORE` which is better than `FLATFILE` for the reasons you mentioned. Now also adding it to `logging.cpp`. Thanks!

(Even with `BLOCKSTORE` I agree it feels a bit bad to add a debug category for this, but I can't find an existing category that would make sense :))